tests/wpt/mozilla/tests/mozilla/media_fragment_seek.html

```diff
@@ -0,0 +1,39 @@
+<!doctype html>
```

Comment:
FYI, there is also a very complete set of tests at https://www.w3.org/2008/WebVideo/Fragments/TC/ua-test-cases in case that you want to port some of them here.

Reply:
Ahh thats good. I incorporated some of those in the test case. I can add more if you would like or another test for invalid semantics.

Reply:
Thanks. Up to you :) More tests can be added as a follow-up.